cpp/bench/ann/src/faiss/faiss_gpu_wrapper.h

```diff
@@ -107,7 +107,7 @@ class FaissGpu : public ANN<T> {
 
   void build(const T* dataset, size_t nrow, cudaStream_t stream = 0) final;
 
-  void set_search_param(const AnnSearchParam& param) override;
+  virtual void set_search_param(const FaissGpu<T>::AnnSearchParam& param) {}
```

Comment:
I'm not too sure about this, should this be?
```suggestion
  virtual void set_search_param(const FaissGpu<T>::AnnSearchParam& param) = 0;
```